tensorflow/python/data/experimental/ops/io.py

```diff
@@ -164,7 +166,8 @@ def element_spec(self):
     return self._element_spec
 
 
-@tf_export("data.experimental.load", v1=[])
+@tf_export("data.experimental.load", "data.load", v1=[])
```

Comment:
`load` should be exposed as a static method on the `tf.data.Dataset` class (e.g. similar to `tf.data.Dataset.zip()`).